USFMToolsSharp/HtmlRenderer.cs

```diff
@@ -118,14 +189,65 @@ private string RenderMarker(Marker input)
                     output.AppendLine(mTMarker.Title);
                     output.AppendLine("</div>");
                     break;
+                case FMarker fMarker:
+                    foreach (Marker marker in input.Contents)
+                    {
+                        RenderMarker(marker);
+                    }
+                    break;
+                case FTMarker fTMarker:
+                    string footnote="";
+                    foreach(Marker marker in input.Contents)
+                    {
+                        switch (marker)
+                        {
+                            case TextBlock textBlock:
+                                footnote += textBlock.Text+" ";
+                                break;
+                            case FQAMarker fQAMarker:
+                                footnote += "<span class=\"footnote-quote\">";
+                                break;
+                            default:
+                                break;
+                        }
+
+                    }
+                    FootnoteTextTags.Add(footnote);
+                    break;
+                case FQAMarker _:
+                    break;
+                case FQAEndMarker fQAEndMarker:
+                    if(FootnoteTextTags.Count > 0)
+                        FootnoteTextTags[FootnoteTextTags.Count-1] += "</span>" + " ";
+                    break;
                 case IDEMarker _:
                 case IDMarker _:
                     break;
+                case VPMarker _:

```

Comment:
There is really no need for the break above this they can all fall through together to the same break.